sympy/core/relational.py

```diff
@@ -1234,6 +1227,13 @@ def is_ge(lhs, rhs):
                 rv = diff.is_extended_nonnegative
                 if rv is not None:
                     return rv
+                n2 = _n2(lhs, rhs)
```

Comment:
Had to make this `_n2()` call here cause there were some expressions like `I**I` which gave `None` during the `.is_extended_nonnegative` check. Will this force evaluate big number in some cases or is the expectation here to competely elimiinate this `_n2()` call?

Reply:
Ideally we would make it so that `n2` is always efficient and then maybe we could use it some of the time. We want to build up alternative ways of resolving these queries that don't depend on n2 though so that we can avoid calling n2 as much as possible.

My suggestion is to remove the n2 and then try to see if there are other ways to fix the things that break as a result. (Basically run the tests, see what fails and try to fix it)